src/Microsoft.DotNet.Build.CloudTestTasks/UploadClient.cs

```diff
@@ -69,11 +70,9 @@ public string EncodeBlockIds(int numberOfBlocks, int lengthOfId)
 
                     if (nextBytesToRead != read)
                     {
-                        log.LogError(
-                            "Number of bytes read ({0}) from file {1} isn't equal to the number of bytes expected ({1}) .",
-                            read,
-                            fileName,
-                            nextBytesToRead);
+                        throw new Exception(string.Format(
+                            "Number of bytes read ({0}) from file {1} isn't equal to the number of bytes expected ({2}) .",
```

Comment:
Don't HAVE to fix this but I have recently fallen in love with $"" formatting...really improves readability.